modules/hcp-virt-reqs.adoc

```diff
@@ -8,8 +8,8 @@
 
 As you prepare to deploy {hcp} on {VirtProductName}, consider the following information:
 
-* Run the hub cluster and workers on the same platform for {hcp}.
-* Each hosted cluster must have a cluster-wide unique name. A hosted cluster name cannot be the same as any existing managed cluster in order for {mce-short} to manage it.
+* The management cluster and workers must both run on {VirtProductName}.
```

Comment:
The management cluster can't be on Virt, otherwise it would be nested virtualization which we don't support. It needs to be on Baremetal, especially if the management clsuter is where the VMs will be located (not virt external mode).

Reply:
Makes sense for OpenShift Virtualization :)